src/game/engine/internals/Rigidbody.js

```diff
@@ -0,0 +1,61 @@
+import { Vector } from '../../shared';
+
+const GRAVITY_ACCELERATION = 0.001;
+
+class Rigidbody {
+  resultVector = Vector.Zero;
+
+  gravityVector = Vector.Zero;
+
+  velocityVector = Vector.Zero;
+
+  accelerationVector = Vector.Zero;
+
+  gravityInit = 0;
```

Comment:
Można by traktować grawitację jako zwykła siła (czyli dla nas przyspieszenie bo zakładamy że każde ciało ma m = 1) i z `addGravity` zrobić coś w stylu `this.accelerationVector = this.accelerationVector.add(new Vector(0, -gravity))`, będziesz musiała się w każdym miejscu przejmować jednym wektorem mniej.